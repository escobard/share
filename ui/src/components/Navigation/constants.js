
export const items = [{ key: "home", active: true, name: "share v1" }];

