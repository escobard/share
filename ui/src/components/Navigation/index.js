import React from "react";
import { Menu } from "semantic-ui-react";

import { items } from "./constants"

// needs to all sections as anchor links
// look for css highlight options when section is selected
const Navigation = () => <Menu items={items} />;

export default Navigation;