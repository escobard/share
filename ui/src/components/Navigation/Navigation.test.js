
import React from 'react';
import { shallow } from 'enzyme';
import Navigation from './index';

describe('Navigation snapshot renders', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<Navigation debug />);

        expect(component).toMatchSnapshot();
    });
});