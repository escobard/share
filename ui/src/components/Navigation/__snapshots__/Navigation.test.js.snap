// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation snapshot renders should render correctly mode 1`] = `
<Menu
  items={
    Array [
      Object {
        "active": true,
        "key": "home",
        "name": "share v1",
      },
    ]
  }
  widths={1}
/>
`;
